go.mod

```diff
@@ -27,12 +28,12 @@ require (
 	github.com/rs/cors v1.7.0
 	github.com/sasha-s/go-deadlock v0.2.0
 	github.com/snikch/goodman v0.0.0-20171125024755-10e37e294daa
-	github.com/spf13/cobra v1.1.0
+	github.com/spf13/cobra v1.1.1
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.1
-	github.com/tendermint/tendermint v0.34.0-rc4
-	github.com/tendermint/tm-db v0.6.2
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
-	golang.org/x/net v0.0.0-20200813134508-3edf25e44fcc
-	google.golang.org/grpc v1.32.0
+	github.com/tendermint/tendermint v0.34.0
```

Comment:
It's a bit odd that tendermint is a dep here. I think this is due to a test.

Reply:
will groom through to find out why it's still here.